src/components/projectTile/ProjectTile.tsx

```diff
@@ -0,0 +1,36 @@
+import placeholder from '../../assets/placeholder_project.png';
+import { Project } from '../../types/Scene';
+import Card from '../card/Card';
+import './ProjectTile.css';
+
+interface ProjectTileProps {
+  project: Project;
+  onClick: () => void;
+}
+
+const ProjectTile = ({ project, onClick }: ProjectTileProps): JSX.Element => (
+  <Card className="project-card">
+    <div onClick={onClick} role="button" tabIndex={0} className="project-wrapper">
+      <img
+        className="project-thumbnail"
+        src={project.thumb || placeholder}

```

Comment:
that's probably out of scope of this PR, but: if thumbnail URL is provided, but invalid (cannot fetch an image), the image changes its size and the whole tile gets smaller